cms/settings/base.py

```diff
@@ -946,3 +949,42 @@
 
 SEARCH_API_DEFAULT_PAGE_SIZE = int(os.getenv("SEARCH_API_DEFAULT_PAGE_SIZE", "20"))
 SEARCH_API_MAX_PAGE_SIZE = int(os.getenv("SEARCH_API_MAX_PAGE_SIZE", "500"))
+
+# Auth
+WAGTAIL_CORE_ADMIN_LOGIN_ENABLED = env.get("WAGTAIL_CORE_ADMIN_LOGIN_ENABLED", "false").lower() == "true"
+LOGOUT_REDIRECT_URL = env.get("LOGOUT_REDIRECT_URL", WAGTAILADMIN_LOGIN_URL)
+AUTH_TOKEN_REFRESH_URL = env.get("AUTH_TOKEN_REFRESH_URL")
+SESSION_COOKIE_AGE = env.get("SESSION_COOKIE_AGE", 60 * 15)  # 15 minutes to match Auth Service
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+IDENTITY_API_BASE_URL = env.get("IDENTITY_API_BASE_URL")
+AWS_COGNITO_LOGIN_ENABLED = env.get("AWS_COGNITO_LOGIN_ENABLED", "false").lower() == "true"
+AWS_COGNITO_USER_POOL_ID = env.get("AWS_COGNITO_USER_POOL_ID")
+AWS_COGNITO_APP_CLIENT_ID = env.get("AWS_COGNITO_APP_CLIENT_ID")
+
+# TODO: Make system checks
+# if AWS_COGNITO_LOGIN_ENABLED:
+#     required_env_vars = [
+#         "IDENTITY_API_BASE_URL",
+#         "AUTH_TOKEN_REFRESH_URL",
+#         "AWS_COGNITO_APP_CLIENT_ID",
+#         "AWS_COGNITO_USER_POOL_ID",
+#     ]
+#     for var in required_env_vars:
+#         if not env.get(var):
+#             raise ImproperlyConfigured(f"Missing required environment variable: {var}")
```

Comment:
@sanjeevz3009 this is left to do

Reply:
I was going to do this as a follow-up PR straight after this and write some tests for it too. Thought it would be cleaner and easier to get this in first and do a follow-up straight after.

Reply:
https://jira.ons.gov.uk/browse/CMS-528